addons/web/static/src/js/core/dom.js

```diff
@@ -539,28 +539,32 @@ var dom = {
     /**
      * @param {HTMLElement} el - the element to stroll to (limitation: if the
      *      element is using a fixed position, this function cannot work except
-     *      if is the header (with the "top" id) or the footer (with the
-     *      "bottom" id) for which exceptions have been made)
+     *      if is the header (el is then a string set to '#top') or the footer
+     *      (el is then a string set to '#bottom') for which exceptions have
+     *      been made)
      * @param {number} [options] - same as animate of jQuery
      * @param {number} [options.extraOffset=0]
      *      extra offset to add on top of the automatic one (the automatic one
      *      being computed based on fixed header sizes)
      * @param {number} [options.forcedOffset]
      *      offset used instead of the automatic one (extraOffset will be
      *      ignored too)
+     * @param {boolean} [options.isTopOrBottom]
+     *      if set, the target of the scroll is the top or the bottom of the
+     *      document depending on the value of el (either '#top' or '#bottom')
```

Comment:
This should not be needed: you ask the caller to give two times the same information. Just check if you received the top or the bottom string inside the util.

Reply:
Yes, that was something I hesitated about in the older version of the PR: At the function call, we already knew if we gave a string as argument or not so it was to avoid having to recheck it in the `scrollTo` function. But yes, the cost was to add one optional argument :)